modules/manager/controller_setup.go

```diff
@@ -508,13 +509,17 @@ func SetupControllers(mgr manager.Manager, c *Config) (map[string]ControllerDef,
 			return nil, err
 		}
 
+		// REVIEW: Should we define the recorder here, or define it out of the section to allow setting custom metrics in other controllers?
+		metricRecorder := metrics.NewGlobalCtrlRuntimeMetricsRecorder()
```

Comment:
I believe we can add it to the top of this func definition as suggested

Reply:
Done.